cmd/minishift/cmd/start.go

```diff
@@ -187,13 +189,21 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
+	automountHostfolders(host.Driver)
```

Comment:
Really we need to look into issue #409 sooner or later. _start.go_ becomes a bit pile of mud :-(